zebrad/tests/acceptance.rs

```diff
@@ -3217,3 +3217,118 @@ async fn trusted_chain_sync_handles_forks_correctly() -> Result<()> {
 
     Ok(())
 }
+
+/// Test successful block template submission as a block proposal on a custom Testnet.
+///
+/// This test can be run locally with:
+/// `RUSTFLAGS='--cfg zcash_unstable="nu6"' cargo test --package zebrad --test acceptance --features getblocktemplate-rpcs -- nu6_lockbox_funding_stream --exact --show-output`
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(all(feature = "getblocktemplate-rpcs", zcash_unstable = "nu6"))]
+async fn nu6_lockbox_funding_stream() -> Result<()> {
```

Comment:
This doesn't actually test what we want yet, I'll open another PR to add the funding stream height ranges to `testnet::Parameters` so it works as expected.